### PR TITLE
Formula for jfrog-cli-go version 1.1.0

### PR DESCRIPTION
**Pull request by eyalbe4**
- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document? Yes.
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change? Yes.
- [ ] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)? Yes.
- [ ] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)? Yes.
### Description

Formula for jfrog-cli-go version 1.1.0.

### CODE REVIEW

File: Formula/jfrog-cli-go.rb [hunk 1]
```diff
@@ -31,8 +31,8 @@ def install
   end
 
   test do
-    actual = pipe_output("#{bin}/jfrog rt -v")
-    expected = "jfrog version 1.0.1\n"
+    actual = pipe_output("#{bin}/jfrog -v")
+    expected = "jfrog version 1.1.0\n"
     assert_equal expected, actual
```
Comment (mikemcquaid) [inline Formula/jfrog-cli-go.rb]: Change this to: `assert_equal "jfrog version #{version}", shell_output("#{bin}/jfrog rt -v").chomp`. If that fails `brew test` locally you may need to add a second argument to `shell_output` to specify the exit code.

Comment (eyalbe4) [inline Formula/jfrog-cli-go.rb]: @mikemcquaid, done.

File: Formula/jfrog-cli-go.rb [hunk 2]
```diff
@@ -31,8 +31,8 @@ def install
   end
 
   test do
-    actual = pipe_output("#{bin}/jfrog rt -v")
-    expected = "jfrog version 1.0.1\n"
+    actual = shell_output("#{bin}/jfrog -v").chomp
```
Comment (mikemcquaid) [inline Formula/jfrog-cli-go.rb]: `assert_equal "jfrog version 1.1.0", shell_output("#{bin}/jfrog -v").chomp` makes this a one-liner so would be preferable

Comment (eyalbe4) [inline Formula/jfrog-cli-go.rb]: @mikemcquaid, done.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `jfrog-cli-go 1.1.0`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**